public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -20,11 +20,11 @@ export function AppChrome({ children }: Props) {
   const { chrome } = useGrafana();
   const state = chrome.useState();
 
-  if (!config.featureToggles.topnav || config.isPublicDashboardView) {
+  if (!config.featureToggles.topnav) {
     return <main className="main-view">{children}</main>;
   }
 
-  const searchBarHidden = state.searchBarHidden || state.kioskMode === KioskMode.TV;
+  const searchBarHidden = state.searchBarHidden || state.kioskMode === KioskMode.TV || config.isPublicDashboardView;
```

Comment:
this won't work as when the DashboardPage loads and wants to add time range controls it will try to add it to an app chrome that is not there